hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -61,6 +63,10 @@
   private Queue<DoubleBufferEntry<OMClientResponse>> currentBuffer;
   private Queue<DoubleBufferEntry<OMClientResponse>> readyBuffer;
 
+
+  private Queue<CompletableFuture<Void>> currentFutureQueue;
```

Comment:
Can you add a one-line comment for the new fields? 

Also we should probably add for the existing fields. I think some ASCII art description of how double buffer works will be helpful to future maintainers. However it's okay to file a follow up jira and do separately later. Don't need to do it for this commit.

Reply:
Opened jira for this.
https://issues.apache.org/jira/browse/HDDS-1874